redaxo/src/core/lib/util/version.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+class rex_version
+{
+    /**
+     * Constructor.
+     */
+    private function __construct()
+    {
+        // noop
+    }
+
+    public static function isUnstable(string $version): bool
+    {
+        foreach (['dev', 'beta', 'alpha', 'rc'] as $unstable) {
+            if (false !== stripos($version, $unstable)) {
+                return true;
```

Comment:
Von der `version_compare`-Logik her ist eigentlich alles unstable, was Text enthält.
Denn dort gilt:
`any string not found in this list < dev < alpha = a < beta = b < RC = rc < # < pl = p`

Zumindest "b" für "beta" habe ich glaube ich schon gesehen als Version bei Addons.

https://www.php.net/manual/en/function.version-compare.php